modules/twinklewarn.js

```diff
@@ -1365,48 +1364,43 @@ Twinkle.warn.callbacks = {
 		var params = pageobj.getCallbackParameters();
 		var messageData = params.messageData;
 
-		var history_re = /<!-- Template:(uw-.*?) -->.*?(\d{1,2}:\d{1,2}),( \d{1,2} \w+ \d{4}) \(UTC\)/g;
+		var history_re = /<!-- Template:(uw-.*?) -->.*?(\d{1,2}:\d{1,2}, \d{1,2} \w+ \d{4} \(UTC\))/g;
```

Comment:
Just a better/different fix than #676, yeah?

Reply:
Just wanted to put the improved constructor to some use, lest a future maintainer were to remove those parts as unused!